addons/stock_picking_batch/report/report_picking_batch.xml

```diff
@@ -26,7 +26,7 @@
                                     </tr>
                                 </thead>
                                 <tbody>
-                                    <tr t-foreach="o.picking_ids" t-as="pick">
+                                    <tr t-foreach="o.picking_ids.sorted(lambda l: l.location_id.display_name)" t-as="pick">
```

Comment:
Why would we need this order here ? The locations are not even displayed on the report, I'm not sure it would really help the users

Reply:
@adwid It's not essential: it generally ensures the pickings are listed in the same order as their associated lines in the detail per location.